skaffold/src/main/kotlin/com/google/container/tools/skaffold/SkaffoldFiles.kt

```diff
@@ -35,7 +35,7 @@ private val SKAFFOLD_API_HEADER_PATTERN: Pattern by lazy {
 /**
  * Checks if a given file is a valid Skaffold configuration file based on type and API version.
  */
-internal fun isSkaffoldFile(file: VirtualFile): Boolean {
+fun isSkaffoldFile(file: VirtualFile): Boolean {
```

Comment:
Question for Kotlin reviewer - we cannot make this work as `internal` with test code (default test code by Gradle conventions), although it should work according to documentation. Can you please hint us where the issue could be?

Reply:
I haven't seen this causing issues just yet. Agreed that, according to docs this should work.

Reply:
I guess we will need to investigate this further, I suspect this has something to do with Gradle sub-modules. Thanks!

Reply:
Filed #43